src/components/Modals/Onboarding.vue

```diff
@@ -528,6 +570,9 @@ export default {
     },
     removeMajor() {
       this.displayOptions.major.pop();
+      if (this.displayOptions.major.length === 0) {
+        this.addMajor();
```

Comment:
Can you change this.addMajor to this.addDefaultMajor and the same for minor? Want these method names to be clear because I didn't realize what they were when we coded this.